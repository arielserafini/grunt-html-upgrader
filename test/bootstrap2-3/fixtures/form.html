<form class="form-horizontal">
  <input id="check" type="checkbox" name="check">
  <div class="control-group">
    <label class="control-label" for="nameInput">Name</label>
    <div class="controls">
      <div class="input-append">
        <input id="nameInput" type="text" name="name">
        <a class="btn"><i class="icon-remove"></i></a>
      </div>
    </div>
  </div>
  <div class="control-group">
    <label class="control-label" for="amountInput">Amount</label>
    <div class="controls">
      <div class="input-prepend">
        <span class="add-on">$</span>
        <input id="amountInput" type="text" name="amount">
      </div>
    </div>
  </div>
  <div class="control-group">
    <label class="control-label" for="descriptionTextarea">Description</label>
    <div class="controls">
      <textarea id="descriptionTextarea" name="description"></textarea>
    </div>
  </div>
</form>

<form>
  <div class="control-group">
    <label class="control-label" for="nameInput">Name</label>
    <div class="controls">
      <input id="nameInput" type="text" name="name">
    </div>
  </div>
  <div class="control-group">
    <label class="control-label" for="descriptionTextarea">Description</label>
    <div class="controls">
      <textarea id="descriptionTextarea" name="description"></textarea>
    </div>
  </div>
</form>
