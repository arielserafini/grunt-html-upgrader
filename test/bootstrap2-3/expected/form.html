<form class="form-horizontal">
  <input id="check" type="checkbox" name="check">
  <div class="form-group">
    <label class="control-label col-lg-3 col-md-3" for="nameInput">Name</label>
    <div class="col-lg-9 col-md-9">
      <div class="input-group">
        <input id="nameInput" type="text" name="name" class="form-control">
        <div class="input-group-btn"><a class="btn btn-default"><i class="icon-remove"></i></a></div>
      </div>
    </div>
  </div>
  <div class="form-group">
    <label class="control-label col-lg-3 col-md-3" for="amountInput">Amount</label>
    <div class="col-lg-9 col-md-9">
      <div class="input-group">
        <span class="input-group-addon">$</span>
        <input id="amountInput" type="text" name="amount" class="form-control">
      </div>
    </div>
  </div>
  <div class="form-group">
    <label class="control-label col-lg-3 col-md-3" for="descriptionTextarea">Description</label>
    <div class="col-lg-9 col-md-9">
      <textarea id="descriptionTextarea" name="description" class="form-control"></textarea>
    </div>
  </div>
</form>

<form>
  <div class="form-group">
    <label class="control-label" for="nameInput">Name</label>
    <div class="controls">
      <input id="nameInput" type="text" name="name" class="form-control">
    </div>
  </div>
  <div class="form-group">
    <label class="control-label" for="descriptionTextarea">Description</label>
    <div class="controls">
      <textarea id="descriptionTextarea" name="description" class="form-control"></textarea>
    </div>
  </div>
</form>
